cli/user/README.md

```diff
@@ -94,7 +95,52 @@ All available commands:
 	stream read scoped-stream-name [group-similar] [timeout-in-seconds]: Reads all Events from a Stream and then tails the Stream.
 ```
 
+You can try out commands such as:
+```
+> scope create example
+Scope 'example' created successfully.
+```
+
 For more info on how the Pravega User CLI works, please visit [this page](https://github.com/pravega/pravega/wiki/Pravega-User-CLI).
 
+## Pravega CLI on Kubernetes
+The pravega-cli can be accessed through the kubernetes pod such as the segmentstore pod.
```

Comment:
Here and in the other document, we may want to keep names starting with capitals: Kubernetes, Bookkeeper, Segment Store, etc.